backends/qualcomm/CMakeLists.txt

```diff
@@ -199,11 +200,6 @@ target_link_libraries(
 #
 target_link_options_shared_lib(qnn_executorch_backend)
 
-#
-# add compile option
-#
-target_compile_options(executorch PUBLIC -DET_EVENT_TRACER_ENABLED)
```

Comment:
@shewu-quic you added this line in https://github.com/pytorch/executorch/pull/2227/files#diff-0f6d37c62838592cf30db121c46cf34a9b9316df4b5a0d98f0ad7c7a98f7ff7eR261

It's currently causing problems because `ET_EVENT_TRACER_ENABLED` only works when `EXECUTORCH_BUILD_DEVTOOLS` is also enabled. Is it ok to remove this, or will other scripts/docs need to change?

Reply:
It should be automatically set by the main `CMakeLists.txt` as long as the CMake flag `EXECUTORCH_ENABLE_EVENT_TRACER` is enabled.